relay-server/src/services/processor.rs

```diff
@@ -1519,6 +1519,7 @@ impl EnvelopeProcessorService {
 
         let global_config = self.inner.global_config.current();
         let ai_model_costs = global_config.ai_model_costs.clone().ok();
+        let ai_operation_type_map = global_config.ai_operation_type_map.clone().ok();
```

Comment:
This should work without cloning (line above as well):

```suggestion
        let ai_operation_type_map = global_config.ai_operation_type_map.as_ref().ok();
```